### ISSUE TITLE
Resource usage graphs for each pod in pod list view.

### ISSUE THREAD

**Issue by aodpi** [lensapp/lens#657]
**What would you like to be added**:
I see that newer version of kubernetes dashboard has resource usage for pods in the pods list view. I think it would be nice to have those graphs with cpu/memory usage for pods list view since it's quite useful. Now I click on each pod to view it's resource usages. I remember that kubernetes dashboard didn't have that previously, but when I updated to version 1.17 this feature was introduced. I believe that there would be compatibility issues with this, although I'm not really sure.

Example image (from kubernetes dashboard):

![image](https://user-images.githubusercontent.com/6562956/89640873-28022500-d8b9-11ea-916f-f633d364687e.png)

**Why is this needed**:
Not so required but good to have. It's a nice and useful enhancement.

**Environment you are Lens application on:**

- Kubernetes distribution: AKS
- Desktop OS: Windows 10 2004

**Comment by j-zimnowoda** [comment]
Very important feature. 
It happens that all pods (containers) are in running state but they still are malfunctioning and consuming way too many resources. Having that view would help discovering potential pod issues.

**Comment by Morriz** [comment]
Yeah, that, plus the fact that it does not refresh data fast enough, makes me stick with `k9s`. What is ttl on cached pod data?

**Comment by jakolehm** [comment]
I think we could close this one because cpu/memory metrics are now part of the pod list view: 
<img width="2032" alt="image" src="https://github.com/user-attachments/assets/38bffdd4-5777-4013-b7a3-4402a958b235">
